CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v2308
+* Feature: Filter findings by last active transport request
+* Fix: SAP Code Inspector findings were missing
+* Fix: SAP Code Inspector finding had wrong code locations
```

Comment:
**SAP Code Inspector finding had wrong code location**

or 

**SAP Code Inspector findings had wrong code locations**

Reply:
Used the second 👍